modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/portlet/AssetPublisherPortlet.java

```diff
@@ -119,185 +140,6 @@
 )
 public class AssetPublisherPortlet extends MVCPortlet {
 
-	public void getFieldValue(
```

Comment:
Hey @MrJohn1911, first make the methods from `AssetPublisherPortlet` private and then, in another commit, move methods from `AssetRSSHelper` to here.